perllib/FixMyStreet/Cobrand/brent.pm

```diff
@@ -0,0 +1,146 @@
+package FixMyStreet::Cobrand::Brent;
```

Comment:
You'll need to keep this (but it has the wrong name) for the cobrand to work properly. But its contents are not correct, yes, it can be cut down to the basics.